app/Platform/RouteServiceProvider.php

```diff
@@ -56,7 +57,7 @@ protected function mapWebRoutes(): void
             // Route::get('create', CreateController::class)->name('create');
             // Route::resource('developers', DeveloperController::class)->only('index');
 
-            // Route::get('docs/api', [ApiDocsController::class, 'index'])->name('docs.api');
+            Route::get('docs/api', [ApiDocsController::class, 'index'])->name('docs.api');
```

Comment:
Is there something I need to run to refresh this? I get a 403 Forbidden error when I click on the API link in the footer.
![image](https://user-images.githubusercontent.com/32680403/230750780-4e4ea4b9-f9c5-4119-98b7-93e841a5dc02.png)

I've already manually run the post-merge commands.
```
php composer.phar install
npm install
npm run build
```

Reply:
With Sail it automagically worked, however I am unsure about other setups. @luchaos do you have any ideas?

Reply:
There might be a conflict with the existing `public/docs` symlink - probably something that has to be fixed in `public/.htaccess` for Apache? 🤔

If that turns out to be a problem on staging, too, then we should probably remove the symlink for now to make sure that URL is routed to the framework.

Reply:
Looks like a broken symlink:
```
Symbolic link not allowed or link target not accessible: /home/brian/source/RAWeb/public/docs
```
```
public/docs -> ../storage/app/static/docs/
```
```
storage/app/static/docs -> ../../../docs/dist/
```
There is no `docs/dist` at the root level.

Running `php artisan ra:storage:link --relative` as mentioned in the README.md did not fix the symlink chain.

Removing the public/docs symlink appears to fix the issue.

Running `php artisan ra:storage:link --relative` again restores the link, thus breaking it again.

See https://github.com/RetroAchievements/RAWeb/blob/42023be338b7251f05bc272bfce9ee3a01e20a9f/app/Support/Filesystem/LinkStorage.php#L38

Reply:
> If that turns out to be a problem with the server's nginx setup, too, then we should probably remove the symlink for now

It doesn't look like there's a docs symlink in production:
```
jamiras@retroachievements:/srv/web/current$ ls -l public/d*
-rw-r--r-- 1 deploy-web www-data  5227 Mar 12 18:23 public/developerstats.php
-rw-r--r-- 1 deploy-web www-data 12619 Mar 12 18:23 public/dorequest.php
-rw-r--r-- 1 deploy-web www-data  1174 Mar 12 18:23 public/doupload.php
-rw-r--r-- 1 deploy-web www-data  3251 Mar 12 18:23 public/download.php
```
```
$ ls -l public/d*
-rw-rw-r-- 1 brian brian  5227 Feb  8 14:34 public/developerstats.php
lrwxrwxrwx 1 brian brian    27 Apr  8 20:37 public/docs -> ../storage/app/static/docs/
-rw-rw-r-- 1 brian brian 12419 Mar 24 10:14 public/dorequest.php
-rw-rw-r-- 1 brian brian  1174 Oct  2  2022 public/doupload.php
-rw-rw-r-- 1 brian brian  3251 Mar  5 10:19 public/download.php
```

Reply:
I have a workaround, and it doesn't seem like the server should be affected (we'll find out shortly). I'm going to proceed with this as-is and the symlink can be fixed in a future PR.